docs/best-practices/ davinci/index.md

```diff
@@ -0,0 +1,164 @@
+
+```hcl
+resource "davinci_flow" "test-flow-1" {
+  environment_id = pingone_environment.my_environment.id
+
+  flow_json = file("./flows/full-minimal.json")
+}
+```
+
+#### Best practice
+
+The below `davinci_flow`<a href="https://registry.terraform.io/providers/pingidentity/davinci/latest/docs/resources/flow" target="_blank">:octicons-link-external-16:</a> definition is best practice (where there is only one connector node in this particular flow), as it does define it's dependent connections in the `connection_link`<a href="https://registry.terraform.io/providers/pingidentity/davinci/latest/docs/resources/flow#nestedblock--connection_link" target="_blank">:octicons-link-external-16:</a> block mapping.  The `errorConnector` connector is also explicitly defined with the `davinci_connection`<a href="https://registry.terraform.io/providers/pingidentity/davinci/latest/docs/resources/connection" target="_blank">:octicons-link-external-16:</a> resource, meaning there will be a consistent configuration when promoting up through test and production.
+
+```hcl
+resource "davinci_connection" "test-error" {
+  environment_id = pingone_environment.my_environment.id
+  connector_id   = "errorConnector"
+  name           = "my-new-error"
+}
+
+resource "davinci_flow" "test-flow-1" {
+  environment_id = pingone_environment.my_environment.id
+
+  flow_json = file("./flows/full-minimal.json")
+
+  connection_link {
+    id                           = davinci_connection.test-error.id
+    name                         = davinci_connection.test-error.name
+    replace_import_connection_id = "53ab************************1ac8"
+  }
+}
+```
+
+### Remove Example / Bootstrapped Configuration from Existing Environments
+
+When following best practices in this section, there may be occasions where example / bootstrapped configuration is present within the environment but is not actively used to fulfil any use cases.  This "orphaned configuration" should be removed and use-cases retested so the configuration does not confuse any audit activities, and to prevent it's accidental use at a later date.
+
+## HCL Writing Recommendations
+
+### Validate `davinci_flow`<a href="https://registry.terraform.io/providers/pingidentity/davinci/latest/docs/resources/flow" target="_blank">:octicons-link-external-16:</a> Terraform HCL before Plan and Apply
+
+When writing Terraform HCL, it's recommended to use `terraform validate` before running `terraform plan` and `terraform apply` when the HCL includes `davinci_flow`<a href="https://registry.terraform.io/providers/pingidentity/davinci/latest/docs/resources/flow" target="_blank">:octicons-link-external-16:</a> resources.
```

Comment:
HCL, **it is** recommended

use of contractions is sometimes problematic for non-native English speakers